pkg/operation/seed/seed.go

```diff
@@ -883,8 +884,8 @@ func bootstrapComponents(c kubernetes.Interface, namespace string, imageVector i
 	}
 	components = append(components, seedadmission.New(c.Client(), namespace, gsacImage.String(), kubernetesVersion))
 
-	// gardener-seed-scheduler
-	schedulerImage := &imagevector.Image{}
+	// gardener-seed-scheduler. Need dummy image in order to allow debootstrap to work
+	schedulerImage := &imagevector.Image{Name: "dummy", Repository: "dummy", Tag: pointer.StringPtr("dummy")}
```

Comment:
Hm, TBH, I don't like it too much (rather not at all).  Can't we rather improve the validations to make instantiating a new object work with a `non-nil` image? Or, simply remove this entirely:

https://github.com/gardener/gardener/blob/4769e2b102502312967fc9d1c1ab8983eb767104/pkg/operation/seed/scheduler/gardener_kube_scheduler.go#L86-L88

This check isn't too meaningful anyways. If it's empty it doesn't work, yes, but for the validation to be really valuable you'd also have to check that a correct image is used. Hence, Maybe we should just get rid of it entirely instead of such workarounds?